odo/tests/test_append.py

```diff
@@ -12,3 +12,16 @@ def test_append_list_to_set():
     s = set([1, 2, 3])
     append(s, [4, 5, 6])
     assert s == set([1, 2, 3, 4, 5, 6])
+
+
+def test_append_set_to_list():
+    s = set([3, 4, 5])
+    lst = [1, 2, 3]
+    append(lst, s)
+    assert lst == [1, 2, 3, 3, 4, 5]
```

Comment:
do we know that this will always be

```python
[1, 2, 3, 3, 4, 5]
```
?

Could it ever end up being 

```python
[1, 2, 3, 4, 3, 5]
```
?

I think a test like this will be immune to this problem:

```python
s = set([3, 4, 5])
lst = [1, 2, 3]
newlist = lst[:]
append(lst, s)
newlist.extend(s)
assert lst == newlist
```